relayer/src/link/operational_data.rs

```diff
@@ -109,21 +166,24 @@ impl OperationalData {
             None => self.batch.iter().map(|gm| gm.msg.clone()).collect(),
         };
 
+        let tm = TrackedMsgs::new(msgs, &self.tracking_id);
+
         info!(
-            "[{}] assembled batch of {} message(s)",
+            "[{}] assembled batch of {} msgs",
```

Comment:
Why abbreviate in this INFO level message?

Reply:
No particular reason (maybe sloppiness). Feel free to fix.